analyses/mwi/dryspells_trigger/exploration/test_arc2_processing.md

```diff
@@ -47,7 +47,7 @@ arc2_test = DrySpells(
 
```

Comment:
Yaaay this notebook runs for me now as well! 
Can we remove this notebook then? Possibly add an issue that we should make a test when porting to toolbox to check if it is working if the file already exists (though we might change the structure anyway)